static/src/javascripts/projects/common/modules/commercial/dfp-api.js

```diff
@@ -519,7 +523,9 @@ define([
         },
         allAdsRendered = function (slotId) {
             if (slots[slotId] && !slots[slotId].isRendered) {
+                slots[slotId].isLoading = false;
                 slots[slotId].isRendered = true;
+                slots[slotId].slot = null;
```

Comment:
You're right, it's a bad habit I must get rid of (trying to kill two birds
with one stone).

I wonder if the health team shouldn't schedule code quality reviews. These
things are easier to reason about with a bird's eye view.

That's a lot of birds.

On 13 January 2016 at 16:21, Jimmy Breck-McKye <notifications@github.com>
wrote:

> In static/src/javascripts/projects/common/modules/commercial/dfp-api.js
> <https://github.com/guardian/frontend/pull/11543#discussion_r49612032>:
>
> >                  slots[slotId].isRendered = true;
> > +                slots[slotId].slot = null;
>
> We can raise a new PR immediately after shipping this one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/11543/files#r49612032>.
>



-- 
–Regis

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396